File: WebApp/Apps/Annotations/App/CustomFieldInfo.ts

class FieldInfo {
    Name: string;
    ObjectId: number;
    Bounds: Bounds;
    OcrFieldInfo: OcrFieldInfo;
    OmrFieldInfo: OmrFieldInfo;
    // constructor();
    constructor() {
        this.Name = "";
        this.ObjectId = 0;
        this.Bounds = new Bounds();
        this.OcrFieldInfo = new OcrFieldInfo();
        this.OmrFieldInfo = new OmrFieldInfo();
    }

}

class Bounds {
    Top: number;
    Left: number;
    Height: number;
    Width: number;
    //constructor();
    constructor() {
        this.Top = 0;
        this.Left = 0;
        this.Height = 0;
        this.Width = 0;
    }
}

class OcrFieldInfo {
    Character: boolean;
    Numeric: boolean;
    Data: boolean;
    EnableOCR: boolean;
    EnableICR: boolean;
    Words: boolean;
    CellBoarders: boolean;
    //constructor();
    constructor() {
        this.Character = false;
        this.Numeric = false;
        this.Data = false;
        this.EnableOCR = false;
        this.EnableICR = false;
        this.Words = false;
        this.CellBoarders = false;
    }
}

class OmrFieldInfo {
    Lowest: boolean;
    Low: boolean;
    High: boolean;
    Highest: boolean;
    Auto: boolean;
    WithFrame: boolean;
    WithoutFrame: boolean;
    // constructor();
    constructor() {
        this.Lowest = false;
        this.Low = false;
        this.High = false;
        this.Highest = false;
        this.Auto = false;
        this.WithFrame = false;
        this.WithoutFrame = false;
    }
}

File: WebApp/Models/CustomFieldInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class CustomFieldInfo
    {
        public string Name { get; set; }
        public int ObjectId { get; set; }
        public Bound Bounds { get; set; }
        public OcrFieldInfo OcrFieldInfo { get; set; }
        public OmrFieldInfo OmrFieldInfo { get; set; }
    }
    public class Bound
    {
        public decimal Top { get; set; }

        public decimal Left { get; set; }
        public decimal Height { get; set; }
        public decimal Width { get; set; }
    }

    public class OcrFieldInfo
    {
        public bool Character { get; set; }
        public bool Numeric { get; set; }
        public bool Data { get; set; }
        public bool EnableOCR { get; set; }
        public bool EnableICR { get; set; }
        public bool Words { get; set; }
        public bool CellBoarders { get; set; }
    }
    public class OmrFieldInfo
    {
        public bool Lowest { get; set; }

        public bool Low { get; set; }
        public bool High { get; set; }
        public bool Highest { get; set; }
        public bool Auto { get; set; }
        public bool WithFrame { get; set; }
        public bool WithoutFrame { get; set; }
    }
}